### PR TITLE
eparusov Added last() method in EntityControlTrait

### PR DESCRIPTION
**Pull request by yogyrton**

### CODE REVIEW

File: src/Traits/EntityControlTrait.php [hunk 1]
```diff
@@ -258,6 +258,15 @@ public function first($where = []): ?Model
         return $result;
     }
 
+    public function last(string $column = null, array $where = []): ?Model
+    {
+        $column ??= 'created_at';
```
Comment (DenTray) [inline src/Traits/EntityControlTrait.php:263]:
```suggestion
    public function last(array $where = [], string $column = 'created_at'): ?Model
    {
```

File: src/Traits/EntityControlTrait.php [hunk 2]
```diff
@@ -258,6 +258,15 @@ public function first($where = []): ?Model
...
+
+        return $this->getQuery($where)
+            ->latest($column)
```
Comment (DenTray) [inline src/Traits/EntityControlTrait.php:266]:
```suggestion
        return $this
            ->getQuery($where)
            ->latest($column)
```

**Review by DenTray — COMMENTED** [review]
@yogyrton please cover new logic by test

File: src/Traits/EntityControlTrait.php (hunk 2 above)
Comment (Goodmain) [reply src/Traits/EntityControlTrait.php]: Done

File: src/Traits/EntityControlTrait.php (hunk 1 above)
Comment (Goodmain) [reply src/Traits/EntityControlTrait.php]: Done

**Review by DenTray — APPROVED** [review]

**Pull request merged**